whelk-core/src/main/groovy/se/kb/libris/Normalizers.groovy

```diff
@@ -33,65 +32,67 @@ example:
 
 @Log
 class Normalizers {
-    
+    private static Set<String> loadedTypes = []
```

Comment:
This is shaky because it depends on which order the normalizers below are created

Reply:
Makes perfect sense. Fixed in [0bf743a](https://github.com/libris/librisxl/pull/1174/commits/0bf743a192678ffe42afbd676478f666cd86946d)